src/routes/store/gift-cards/+page.svelte

```diff
@@ -12,16 +16,16 @@
       <main class="flex-1">
         <div class="grid lg:grid-cols-3 grid-cols-1 xl:gap-3 gap-5 mb-8">
 
-          {#each Array(10) as item}
+          {#each gift_cards as item}
             <div class="group overflow-hidden relative flex flex-col bg-white shadow rounded-xl dark:bg-neutral-700/30 dark:border-neutral-700 dark:shadow-neutral-700/70 transition hover:drop-shadow-xl hover:shadow-sm hover:-translate-y-1 hover:translate-x-1 hover:scale-[1.025]">
-              <img class="w-full h-auto rounded-t-xl" src="https://images.unsplash.com/photo-1680868543815-b8666dba60f7?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=320&q=80" alt="Product Thumb">
+              <img class="w-full h-auto rounded-t-xl" src={item.imgUrl} alt={item.name}>
 
               <div class="px-4 md:px-5 py-2 md:py-3 flex-1">
-                <h3 class="text-lg font-bold text-gray-800 dark:text-white border-b-2 pb-3 mb-5">VISA Card</h3>
+                <h3 class="text-lg font-bold text-gray-800 dark:text-white border-b-2 pb-3 mb-5">{item.name}</h3>
 
                 <div class="flex flex-wrap items-start gap-1.5">
-                  {#each Array(10) as item}
-                    <span class="inline-flex items-center py-1 px-2 rounded-full text-xs bg-brand-200 text-brand-900 dark:bg-brand-800/30 dark:text-brand-400">10 USDT</span>
+                  {#each item.price_tags.denominations as denomination}
+                    <span class="inline-flex items-center py-1 px-2 rounded-full text-xs bg-brand-200 text-brand-900 dark:bg-brand-800/30 dark:text-brand-400">{!item.discount ? Math.ceil((denomination*item.price_tags.commission)) : Math.ceil((denomination*item.price_tags.commission)/item.discount) } USDT</span>
                   {/each}
```

Comment:
All prices should be in dollars. Use the toCurrency method.

We will implement USDT and other currency conversions separately and from live APIs later on.